release/assets/macos_cli_installation.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -e
+
+USER_PROVIDED_PATH=$1
+
+main() {
+  cat << EOF
+   ______
+  / ____/___  ____    ( )_  _______
+ / /   / __ \/ __ \  / / / / / ___/
+/ /___/ /_/ / / / / / / /_/ / /
+\____/\____/_/ /_/_/ /\____/_/
+                /___/
+EOF
+
+  if [[ $USER_PROVIDED_PATH = "--help" ]]; then
+    print_help
+  fi
+  executable_path=${USER_PROVIDED_PATH:-'/usr/local/bin'}
+
+  echo "Configuring the Conjur CLI. This will take a few moments...."
+  create_symbolic_link $executable_path
+
+  conjur --help > /dev/null
```

Comment:
what's going on here? why are we doing this?

Reply:
we do that because the first time conjur is being run it takes few seconds. So we do this in the install in a way client dont see it. After first run of CLI it runs quick

Reply:
can we add a comment saying that?

Reply:
sure!